composite-actions/nodejs-script/service-definition-validator/src/utils/templateLoader.ts

```diff
@@ -0,0 +1,31 @@
+import inputApiStaging from "../../templates/input-api/staging.json" with { type: "json" };
```

Comment:
For the next version of this:
1. Should the types be configurable as an input to the action so that ANY service type can be used, not just the PnP types as of now?
2. If yes, then the path to the templates should also be part of that config?
3. Perhaps this defeats our main objective as of now, which is to assert that all services follow the templates. That could be more difficult if templates were scattered around everywhere...?